lib/new_relic/agent/error_filter.rb

```diff
@@ -144,8 +145,16 @@ def log_filter(setting, errors)
         end
       end
 
+      #edit this to account for integers
```

Comment:
I think this might be a leftover from the WIP! Do you think this comment is still needed?

Reply:
Nope! Deleting it.